docs/_edot-sdks/python/setup/manual-instrumentation.md

```diff
@@ -8,4 +8,108 @@ grand_parent: EDOT Python
+
+```sh
+export OTEL_RESOURCE_ATTRIBUTES=service.name=<app-name>
+export OTEL_EXPORTER_OTLP_ENDPOINT=https://my-deployment.apm.us-west1.gcp.cloud.es.io
+export OTEL_EXPORTER_OTLP_HEADERS="Authorization=Bearer P....l"
+```
+
+Refer to our [Setup](./index) page for more details.
+
+<!-- ✅ Manually instrument the application and start sending data to Elastic -->
+### Manually instrument your auto instrumented Python application
+
+In this section we'll show how to add manual instrumentation to an already automatically instrumented application. A use case for
+this setup would be to trace something in particular while keeping the benefits of the simplicity of the automatic instrumentation doing
+the hard work for us.
+
+As an example we'll use an application using the Flask framework that implements an endpoint mounted on `/hello` returning a friendly
+salute. This application is saved in a file named `app.py` that is the default module for Flask applications.
+
+```
+import random
+
+from flask import Flask
+from opentelemetry import trace
+
+tracer = trace.get_tracer(__name__)
+
+app = Flask(__name__)
+
+@app.route("/hello")
+def hello():
+    choices = ["there", "world", "folks", "hello"]
+    # create a span for the choice of the name, this may be a costly call in your real world application
+    with tracer.start_as_current_span("choice") as span:
+        choice = random.choice(choices)
+        span.set_attribute("choice.value", choice)
+    return f"Hello {choice}!"
+```
+
+
+We need to make sure to have Flask and the Flask OpenTelemetry instrumentation installed:
+
+```bash
+pip install flask
+edot-bootstrap --action=install
+```
+
+And then we can run this application with the following command:
+
+```bash
+opentelemetry-instrument flask run
+```
+
+We may not only need to add a custom span to our application but also want to use a custom metric, like in the example below where we
+are tracking how many times we are getting one of the possible choices for our salutes.
+
+```
+import random
+
+from flask import Flask
+from opentelemetry import metrics, trace
+
+tracer = trace.get_tracer(__name__)
+meter = metrics.get_meter(__name__)
+
+hello_counter = meter.create_counter(
+    "hello.choice",
+    description="The number of times a salute is chosen",
+)
+
+app = Flask(__name__)
+
+@app.route("/hello")
+def hello():
+    choices = ["there", "world", "folks", "hello"]
+    # create a span for the choice of the name, this may be a costly call in your real world application
+    with tracer.start_as_current_span("choice") as span:
+        choice = random.choice(choices)
+        span.set_attribute("choice.value", choice)
+    hello_counter.add(1, {"choice.value": choice})
+    return f"Hello {choice}!"
+```
+
+<!--  ✅ What success looks like -->
+## Confirm that EDOT Python is working
+
+To confirm that EDOT Python has successfully connected to Elastic:
+
+1. Go to **APM** → **Traces**.
```

Comment:
This is more like Application -> Service Inventory

![Schermata del 2025-03-20 09-43-59](https://github.com/user-attachments/assets/cdadcc1e-e870-4047-8661-c83247af6bee)